src/components/Display.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => {
+  const {
+    value,
+  } = props;
+  return (
+    <div className="calc-display">
+      {value}
+    </div>
+  );
+};
```

Comment:
[Optional]

Like `Button` component we could also destructure the `value` prop directly from the argument.

```diff
- const Display = (props) => {
+ const Display = ({ value }) => (
-  const {
-    value,
-  } = props;
-  return (
    <div className="calc-display">
      {value}
    </div>
  );
-};
```